client/src/shared/ExecutionInfo.tsx

```diff
@@ -0,0 +1,46 @@
+import { UnifiedGame } from "@hidden-identity/shared";
+import { usePlayerOrder } from "./PlayerListOrder";
+import { Text } from "@radix-ui/themes";
+
+interface ExecutionInfoProps {
+  game: UnifiedGame;
+  firstSeat: string | null;
```

Comment:
Neither of these props should be necessary to this component.
`game` can be acquired via `useDefiniteGame` which holds game in a global context.
`players` can be read from game as `game.playerList` or some similarly named property.